typhon/retrieval/spareice/common.py

```diff
@@ -134,8 +138,8 @@ def __init__(self, file=None, collocator=None, processes=10, verbose=2,
             processes: Number of processes to parallelize the training or
                 collocation search. 10 is the default. Best value depends on
                 your machine.
-            verbose: An integer value. The higher the value, the more debug
-                messages are printed.
+            verbose (int): Control ``GridSearchCV`` verbosity. The higher the
```

Comment:
Here, one could implicitly use the logging level to determine what to pass to ``GridSearchCV``.
```python
level = logger.getEffectiveLevel()
if level >= logging.DEBUG:
    verbose = 2
```
or something like that. I was not sure, if that is needed or not. Therefore, I kept it as close to the existing code as possible.